tests/test_authz_Authorizer.py

```diff
@@ -106,23 +111,59 @@ def test_validate_all(self):
         assert self.authz.outcome == ALLOW
 
     def test_validate_fail_all(self):
```

Comment:
I would indicate this tests checks expiration or add separate method

Reply:
this is correct this check does not check exp validation per se.